engine/src/flutter/lib/ui/fixtures/ui_test.dart

```diff
@@ -238,6 +238,74 @@ void sendSemanticsUpdate() {
   _semanticsUpdate(builder.build());
 }
 
+@pragma('vm:entry-point')
+void sendSemanticsUpdateWithRole() {
+  final SemanticsUpdateBuilder builder = SemanticsUpdateBuilder();
+
+  final Float64List transform = Float64List(16);
+  final Int32List childrenInTraversalOrder = Int32List(0);
+  final Int32List childrenInHitTestOrder = Int32List(0);
+  final Int32List additionalActions = Int32List(0);
+  transform[0] = 1;
+  transform[1] = 0;
```

Comment:
Don't need to zero out initial values, right? they are already `0`

Reply:
@yjbanov also says that if you access the last index first, the compiler will skip bounds checks.

Reply:
I remember the unnecessary 0, and since this is only a test file, i am less worry about performance, but more on readability